tests/flow/test_ts_madd.py

```diff
@@ -130,7 +140,13 @@ def test_madd_some_failed_replicas():
         env.assertEqual(r.execute_command("ts.range", "test_key1", "-", "+"),  [[122, b'11'], [123, b'11'], [124, b'12'], [125, b'12']])
 
     if env.useAof and isinstance(env.envRunner, StandardEnv):
-        cmds = aof_parser.parse_file(os.path.join(env.envRunner.dbDirPath, env.envRunner._getFileName('master', '.aof')))
+        dbDirPath = env.envRunner.dbDirPath
+        if not is_less_than_ver7:
```

Comment:
I am not sure I understand this differentiation to Redis 7, please explain and add a comment.